src/multi.c

```diff
@@ -79,8 +79,15 @@ void queueMultiCommand(client *c, uint64_t cmd_flags) {
     mc = c->mstate.commands+c->mstate.count;
     mc->cmd = c->cmd;
     mc->argc = c->argc;
-    mc->argv = c->argv;
-    mc->argv_len = c->argv_len;
+    if(c->argv == c-> argv_static) {
```

Comment:
```suggestion
    if(c->argv == c->argv_static) {
```

Reply:
```c
#define argvConvert(c) \
 do {\
    if(c->argv == c-> argv_static) {\
         c->argv = zmalloc(sizeof(robj*)*c->argc);\
         c->argv = memcpy(c->argv, c->argv_static, sizeof(robj*)*c->argc);\
    }\
}while(0)\
```
I think all conversions can use this macro define.